Tasks/AzureAppServiceManage/Tests/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "devDependencies": {
-    "nock": "9.0.11"
+    "nock": "9.0.11",
+    "tunnel": "0.0.5"
```

Comment:
was this intentional? or is this because the PR is long running and master is moving forward (i.e. tunnel was removed in master, but you are based on an older branch?)

Reply:
should the bulk of the files be reverted, check in to master, and then separately add the generated files?

with `git checkout VERSION -- PATH` it would be easy to pluck out the handful of files that need to change, and revert the rest.